src/content/energy_consumption/index.md

```diff
@@ -0,0 +1,76 @@
+
+Ethereum has used proof-of-work since genesis. Migrating off of proof-of-work has always been a fundamental goal of Ethereum. Still, it has been philosophically and technologically challenging because the viable alternatives all required to compromise Ethereum's core principles.
+
+## Proof-of-work energy expenditure {#proof-of-work}
+
+Proof-of-work is a robust way to secure against dishonest changes to the blockchain, but it is problematic for several reasons. Since the right to mine a block requires solving a computational puzzle, miners can increase their odds of success by investing in more powerful hardware. These incentives cause an arms race with miners acquiring increasingly power-hungry mining equipment. Ethereum's proof-of-work protocol currently consumes as much energy as a medium-sized country.
+
+## Proof-of-stake {#proof-of-stake}
+
+A greener future for Ethereum is already being built in the form of a **proof-of-stake (PoS)** chain. Under proof-of-stake, arbitrary puzzle-solving is unnecessary. Removing puzzle-solving drastically reduces the energy expenditure required to secure the network. Miners get replaced by validators who perform the same function except that instead of expending their assets up-front in the form of computational work, they stake ETH as collateral against dishonest behavior. If the validator's node is non-responsive or a fraudulent block gets submitted to the chain, the staked assets can be "slashed", strongly incentivizing honesty and securing the network.
+
+Similarly to proof-of-work, to maintain a fraudulent blockchain, a validator would require 51% of the total ETH staked in the network. However, unlike proof-of-work, consensus is not based on the longest chain, but a mechanism known as ["Casper"](https://arxiv.org/abs/1710.09437). Migrating from proof-of-work to proof-of-stake eliminates the need to expend energy on arbitrary computations.
+
+## The merge {#the-merge}
+
+There is a functional proof-of-stake chain called the [Beacon Chain](/eth2/beacon-chain/) that has been running since December 2020 that is demonstrating the viability of the proof-of-stake protocol. The merge refers to the point in time when Ethereum leaves proof-of-work behind and fully adopts proof-of-stake. The merge is expected to happen ~Q2 2022. [More on the merge](/eth2/merge/).
+
+## Proof-of-stake energy expenditure {#proof-of-stake-energy}
+
+As well as building confidence in the proof-of-stake mechanism, the Beacon Chain also enables estimates of Ethereum's post-merge energy usage. A recent [blog post on the ethereum.org blog](https://blog.ethereum.org/2021/05/18/country-power-no-more/) suggested that the merge to proof-of-stake could result in a 99.95% reduction in total energy use, with proof-of-stake being ~2000x more efficient than proof-of-work. The energy expenditure of Ethereum will be roughly equal to the cost of running a home computer for each node on the network.
+
+![image](energy_use_per_transaction.png)
```

Comment:
Question about this image. My understanding was that a block was using around this amount of energy, not a transaction. Am I out to lunch on my understanding? If not, then I think we need a different image.

Reply:
> [Digiconomist](https://digiconomist.net/ethereum-energy-consumption/) estimates that Ethereum miners currently consume 44.49 TWh per year which works out to 5.13 gigawatt on a continuing basis. This means that PoS is ~2000x more energy efficient based on the conservative estimates above, which reflects a reduction of at least 99.95% in total energy use.
>
> If energy consumption per-transaction is more your speed, that’s ~35Wh/tx (avg ~60K gas/tx) or about 20 minutes of TV. By contrast, Ethereum PoW uses the equivalent energy of a house for 2.8 days per transaction and Bitcoin consumes 38 house-days worth.

This seems to be referring to per-transaction cost, at an average gas of 60k. 

But on that note, I think these stats are a bit out-dated:

![image](https://user-images.githubusercontent.com/54227730/139953275-8f028519-1372-4200-adf7-9557957612e3.png)

This shows per-transaction PoW energy usage in Ethereum as 175kWh, vs the 84kWh shown on the image. Not necessarily the end of the world, but we should at least note a timestamp for that image so people understand it's not necessarily fully up-to-date.

Reply:
maybe just state a range with citations - e.g. 

"Estimate of PoW energy consumption used in figure based on [May 2021 data](https://blog.ethereum.org/2021/05/18/country-power-no-more/), at time of writing the same source suggested [175.56 Kwh]((https://digiconomist.net/ethereum-energy-consumption))"

will include in next commit - please delete if you feel it's excessive.